docs/.vuepress/config.ts

```diff
@@ -434,6 +434,10 @@ function getSidebar(locale: string): SidebarOptions {
                   text: "Ethereum Gravatar (Simple)",
                   link: `${locale}/quickstart/quickstart_chains/ethereum-gravatar.md`,
                 },
+                {
+                  text: "Ethereum Uniswap (Medium)",
```

Comment:
```suggestion
                  text: "Ethereum Uniswap (Complex)",
```

Reply:
You think this project is less complicated than the Gravatar one?

Reply:
Personally, I find it more complicated. If you are thinking it isn't, why are you suggesting the opposite in https://github.com/subquery/documentation/pull/415#discussion_r1328833488.

Reply:
@jamesbayly?